index.html

```diff
@@ -14,13 +15,14 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
-      <nav id="desktop-nav">
-        <ul>
+      <nav id="desktop-nav" class="hidden">
+        <ul class="menu pop-up hidden">
+          <li><button class= "cross">X</button></li>
```

Comment:
- [ ] As stated before😃 by the previous reviewer I think you forgot to add the style for the button close💯, so, please add the style for it following the Figma Design😊, so the button will be nice for the user, to see the Figma Design [Click Here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=9MV4O5YHc8akkclq-0)🎈🪄

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/211066626-603bc99d-ee56-49c0-a658-d0593d4ab362.png)   | ![image](https://user-images.githubusercontent.com/83360636/211066851-7dbbbd6e-ec1c-4422-ada3-ef7d12526a19.png) |